jvm-packages/xgboost4j/src/test/java/ml/dmlc/xgboost4j/java/BoosterImplTest.java

```diff
@@ -24,6 +24,166 @@
 import junit.framework.TestCase;
 import org.junit.Test;
 
+import java.util.Random;
+import java.io.PrintStream;
+import java.util.*;
+import java.util.concurrent.*;
+
+//
+// Utility class for comparing array contents
```

Comment:
this class is redundant as you can make use of 
[org.junit.Assert.assertArrayEquals](https://junit.org/junit4/javadoc/latest/src-html/org/junit/Assert.html#line.515)

Reply:
removed